testutil/compose/define.go

```diff
@@ -257,7 +257,7 @@ func buildLocal(ctx context.Context, dir string) error {
 	log.Info(ctx, "Building local charon binary", z.Str("repo", repo), z.Str("target", target))
 
 	cmd := exec.CommandContext(ctx, "go", "build", "-o", target)
-	cmd.Env = append(os.Environ(), "GOOS=linux", "GOARCH=amd64")
+	cmd.Env = append(os.Environ(), "GOOS=linux", "GOARCH=amd64", "CGO_ENABLED=0")
```

Comment:
discussion: why do we need to set `CGO_ENABLED` flag here?

Reply:
when building go programs, it is best practice to explicitly disable CGO if not required, otherwise some C libraries and stuff is included in the build which can impact compatibility.